client/knowledge_base_search/src/containers/ResultContainer/ResultContainer.jsx

```diff
@@ -1,9 +1,36 @@
 import React from 'react';
 import classNames from 'classnames';
 import { string, number, node, oneOfType } from 'prop-types';
+import { Icon } from 'watson-react-components';
 import './styles.css';
 
-function ResultContainer({ resultText, resultRank, children }) {
+function ResultContainer({ resultText, resultRank, originalRank, children }) {
+
+  function getRankDisplay(resultRank, originalRank) {
+    if (resultRank < originalRank) {
+      return (
+        <div className="training_comparison--rank-down">
```

Comment:
would change this to BEM format, `training_comparison` is the "block" (B), `rank` is the "element" (E) and `down` is the "modifier" (M), then use the following comment for the SASS in styles